src/main/java/somethingrandom/entity/Item.java

```diff
@@ -1,4 +1,60 @@
 package somethingrandom.entity;
 
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * An abstract Item.
+ */
 public abstract class Item {
+    /**
+     * The Name.
+     */
+    public String name;
```

Comment:
The fields here (and in `ActionableItem`, maybe others) should ideally be `private`, since we have the getters/setters to allow access to them.